playground/kafka/Consumer/Program.cs

```diff
@@ -6,8 +6,10 @@
 
 var builder = Host.CreateApplicationBuilder(args);
 
+var kafka = args[0];
```

Comment:
none of our samples do this. I would revert to make it consistent with what they are doing.

Reply:
I will do. Is this supported or is it a side effect ?
It brings ability to write the following apphost which I find interesting

```c#
var builder = DistributedApplication.CreateBuilder(args);

var kafka = builder.AddKafka("kafka")
    .WithKafkaUI(kafkaUi => kafkaUi.WithHostPort(8080));

builder.AddProject<Projects.Producer>("producer")
    .WithReference(kafka)
    .WithArgs(kafka.Resource.Name);

builder.AddProject<Projects.Consumer>("consumer")
    .WithReference(kafka)
    .WithArgs(kafka.Resource.Name);

var kafka2 = builder.AddKafka("kafka2").WithKafkaUI();

builder.AddProject<Projects.Producer>("producer2")
    .WithReference(kafka2)
    .WithArgs(kafka2.Resource.Name);

builder.AddProject<Projects.Consumer>("consumer2")
    .WithReference(kafka2)
    .WithArgs(kafka2.Resource.Name);
```

Reply:
@davidfowl reverted